test_scripts/Defects/5_0/2462_Need_updated_logic_for_checking_user_consent_prompt_after_PTU.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2462
+--
+-- Description:
+-- 1) Need updated logic for checking user consent prompt after PTU.
+-- Steps to reproduce:
+-- 1) SDL does not check new added group from PTU requires user consent prompt if it was not 
+--    present in local PT.
+-- Actual:
+-- 1) SDL does not send onAppPermissionChange notification with appPermissionsConsentNeeded set to true 
+--    for affected application. As a result permissions popup is shown only after the next application activation.
+-- Expected:
+-- 1) SDL will be check a new added group from PTU requires user consent prompt if it not present in local PT.
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local functions ]]
+local function expectationAppPermissionsParam()
```

Comment:
expectation must also contain the expectation `m.getHMIConnection():ExpectNotification("SDL.OnStatusUpdate", { status = "UP_TO_DATE" })`